src/store/index.ts

```diff
@@ -210,6 +211,30 @@ const resolvers: Resolvers = {
         }
     },
     Mutation: {
+        createUnownedTrackable: async (_root, { input }: MutationCreateUnownedTrackableArgs, { communityPromise, cache }: TrackerContext): Promise<CreateTrackablePayload | undefined> => {
```

Comment:
do we need both `createTrackable` and `createUnownedTrackable`, or can this just become `createTrackable`?

Reply:
good question :)

Reply:
good question :)

Reply:
my guess is we don't, but I say we merge and adjust the names as we implement more?